src/components/app/details/cicdHistory/LogsRenderer.tsx

```diff
@@ -37,7 +38,12 @@ export default function LogsRenderer({
     ) : (
         <div className="logs__body">
             {logs.map((log, index) => {
```

Comment:
Please add log type

Reply:
This was declared earlier so I didn't edited that. Should I add types!!??

Reply:
Updated the types and also has added min-height for empty logs to maintain symmetry.
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/68757539/216707320-09c2592c-e41c-449e-aadc-366d04bc1938.png">